jme3-plugins/src/gltf/java/com/jme3/scene/plugins/gltf/LightsPunctualExtensionLoader.java

```diff
@@ -207,15 +220,16 @@ private PointLight buildPointLight(JsonObject obj) {
     private void addLight(Node parent, Node node, int lightIndex) {
         if (lightDefinitions.containsKey(lightIndex)) {
             Light light = lightDefinitions.get(lightIndex);
+            light = light.clone();
```

Comment:
This is quite an important change. What kind of problem did you encounter by not cloning the `light` object taken from the `lightDefinitions` map? Could it be that the `lightIndex` is associated with multiple `nodes`, causing inconsistencies? Just curious

Reply:
This is a fix. 
The LightControl cannot properly move a light with its parent spatial, if it shared among multiple spatials.

Enabling https://github.com/jMonkeyEngine/jmonkeyengine/pull/2548  by default in the gltf loader, makes the light control obsolete, so this fix could be avoided, but i am not sure that jme can share light instances, anyway.